core/templates/dev/head/base.html

```diff
@@ -128,12 +128,14 @@
                           <li class="dropdown pull-right">
                             <a class="dropdown-toggle oppia-navbar-profile-dropdown-toggle" data-toggle="dropdown" ng-mouseover="onMouseoverProfilePictureOrDropdown($event)" ng-mouseleave="onMouseoutProfilePictureOrDropdown($event)">
                               <div class="oppia-navbar-profile-picture-container" ng-cloak>
+                                <profile-link-image linked-name="{{username}}">
```

Comment:
Ah I see, so down the line we'd want like an "I'm looking at my own profile page" profile page, not a public one. I think I was thinking of Facebook, where if I click on my name I do in fact go to my profile page, as it appears to all my friends.